server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java

```diff
@@ -698,6 +701,7 @@ public synchronized ScanFileManager newScanFileManager(ScanDispatch scanDispatch
 
     private final AtomicLong lastReportedSize = new AtomicLong();
     private final AtomicLong lastReportedMincSize = new AtomicLong();
+    private final AtomicReference<Timer> firstReportedCommitTimer = new AtomicReference<>(null);
```

Comment:
If I'm understanding this correctly, we are tracking the first time a write is performed to any Tablet in the TabletServer and then flushing the in-memory map when the threshold is crossed for any table? That's because there is only one in-memory map that is used for mutations for all tables, right? I'm wondering if the property should be for the tablet server, not the table.

Reply:
The tracking and flushing is per tablet not per tablet server. This code is inside an inner class named TabletResourceManager that does a bit of per tablet resource management.

Reply:
Each tablet has its own in memory map of mutations.

Reply:
Ok, I missed the inner class. Sorry about that.